docs/authoring.md

```diff
@@ -201,6 +202,11 @@ The `.tp-config.json` file contains a JSON object that may have the following in
     "server": {
       "description": "The server you want to use in the code.",
       "required": true
+    },
+    "pathToApiFile": {
```

Comment:
please add some more explanation to it:
- that when you access this param in your template with {{params.pathToApiFile}} you get the default value if other not provided
- that it will never work with params that are marked as `required` as those you always have to provide explicitly

It is not that obvious when looking at the example

Reply:
you can just extend the description in the table where default param is described and it will be fine

Reply:
Seems I was a little dizzy when wrote this 😃 
Everything make sense, will fix